src/main/java/com/google/sps/tool/EntityTestingTool.java

```diff
@@ -0,0 +1,58 @@
+package com.google.sps.tool;
+
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+
+import java.util.List;
+import java.util.ArrayList;
+import com.google.sps.data.Card;
+import com.google.sps.data.Folder;
+
+public class EntityTestingTool {
```

Comment:
Do we need to create a separate class for this? Why can't these methods live in the test class(es)?

Also, the naming of this class suggests that it would become some sort of "God" util object. Those are usually code smells — there is probably a way to decompose this class into more focused util objects (such as by data model).

Reply:
Thanks for bringing this up. Maybe when I was creating the test servlets, I assumed that I would need a lot of helper methods, which led me to creating a class called EntityTestingTool. Now that I am looking back at my code, I don't have that much util methods. Reinserting the method in the test class(es) would probably make the code readable.